src/lib/executeSourceTransfer.js

```diff
@@ -2,10 +2,17 @@
 
 const log = require('../common').log.create('executeSourceTransfer')
 const validator = require('./validate')
+const moment = require('moment')
+const promiseRetry = require('promise-retry')
+const retryOpts = { // 22 retries over approx. 24 hours
+  retries: 22,
```

Comment:
I think I would just make this something like `Infinity` because we don't really want it to stop retrying because of this. The thing that will make it stop retrying is the `timeoutPromise`